pkg/apis/admin/v1beta2/types.go

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2018, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package v1beta2
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+/*
+ * Be careful with the comments in this file. The prefix "+" indicates that this is being processed
+ * by the client generator. The location, empty lines, and other comments in this file may confuse
+ * the generator, and produce a non-version version.
+ */
+
+// ** AddressPlan
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +genclient:noStatus
+
+type AddressPlan struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec AddressPlanSpec `json:"spec"`
+}
+
+type AddressPlanSpec struct {
+	AddressType      string             `json:"addressType"`
+	DisplayName      string             `json:"displayName,omitempty"`
+	LongDescription  string             `json:"longDescription,omitempty"`
+	ShortDescription string             `json:"shortDescription,omitempty"`
+	DisplayOrder     int                `json:"displayOrder,omitempty"`
+	Resources        map[string]float32 `json:"resources,omitempty"`
```

Comment:
Since its a Double in java, I think this should be float64.

Reply:
Fixed.